packages/salesforcedx-vscode-apex/src/telemetry/index.ts

```diff
@@ -5,4 +5,4 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode';
-export const telemetryService = TelemetryService.getInstance();
+export const telemetryService = TelemetryService.getInstance('salesforcedx-vscode-apex');
```

Comment:
I do not feel it is necessary to specify a name for the core extensions such as apex & lightning. The reason why we pass in the name is only that e4d is using the telemetryService by salesforcedx-vscode-core. However, extensions in this core repo use their own TelemetryService 
`import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode';` makes it.

Reply:
Of course, it does not hurt to use in this way. Actually the name is specified during initializeService for now (not ideally).